life_expectancy/cleaning.py

```diff
@@ -1,32 +1,73 @@
 import argparse
+from abc import ABC, abstractmethod
 import pandas as pd
 from life_expectancy.i_o import load_data, save_data
+from life_expectancy.countries import Country
 
-def clean_data(df):
+class FileCleaner(ABC):
+    """
+    Abstract class for reading different types of files
+    """
+    ext=''
+    @abstractmethod
+    def clean_data(self, df: pd.DataFrame):
+        """
+        Abstract method for reading different types of files
+        """
+        pass
+
+class TsvFileReadClean(FileCleaner):
+    """
+    Class to clean tsv files that inherits from the FileCleaner abstract class
+    """
+    ext='tsv'
+    def clean_data(self, df):
```

Comment:
```suggestion
    ext: str = 'tsv'
    def clean_data(self, df: pd.DataFrame) -> pd.DataFrame:
```
~(>_<。)＼ don't forget the type hints!

Reply:
Same goes for ZippedJsonReadClean

Reply:
Addresed